shared/wallets/routes.js

```diff
@@ -167,19 +167,21 @@ class WalletsSubNav extends React.PureComponent<any> {
   }
 }
 
+const WalletsSubNavigator = createNavigator(
```

Comment:
Mobile gets mad about this
<img width="300" alt="image" src="https://user-images.githubusercontent.com/11968340/55355905-80d8d680-5497-11e9-83e0-130c4035cc66.png">